sotodlib/preprocess/processes.py

```diff
@@ -4,9 +4,12 @@
 import sotodlib.tod_ops as tod_ops
 from sotodlib.hwp import hwp
 
-from sotodlib.core.flagman import (has_any_cuts, has_all_cut,
-                                   count_cuts,
-                                    sparse_to_ranges_matrix)
+from sotodlib.core.flagman import (
```

Comment:
I think our PR best practices is to not include formatting changes in the same PR as substantive code changes.

Reply:
Sorry, sometimes I forget to turn off my `black` hook...